docs/cluster_spec.md

```diff
@@ -55,6 +55,29 @@ spec:
     - 12.34.56.78/32
 ```
 
+### cluster.spec Subnet Keys
+
+#### id
+ID of a subnet to share in an existing VPC.
+
+#### egressId
+The resource identifier (ID) of something in your existing VPC that you would like to use as "egress" to the outside world. This feature was originally envisioned to allow re-use of NAT Gateways so the correct form for this field is as follows.
+
+```
+spec:
+  subnets:
+  - cidr: 10.20.64.0/21
+    name: us-east-1a
+    egressId: nat-987654321
```

Comment:
I do wonder if this should be at cluster level.  But ... this is more powerful, we can make it optional, so I think I agree with you :-)